src/dataset-builder/ConceptSelector.test.ts

```diff
@@ -25,12 +25,12 @@ describe('ConceptSelector', () => {
   // Using 101 so the ID doesn't match the count.
   const rootConcept = { ...dummyGetConceptForId(100), children: [dummyGetConceptForId(101)] };
   const initialCart: SnapshotBuilderConcept[] = [];
-  const renderSelector = () => {
+  const renderSelector = (cart?: SnapshotBuilderConcept[]) => {
```

Comment:
Instead of this approach you can use a default argument
```suggestion
  const renderSelector = (initialCart: SnapshotBuilderConcept[] = []) => {
```
See `ConceptSearch.test.ts` for an example of this.